.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -34,5 +34,10 @@ If applicable, add screenshots to help explain your problem.
  - Browser: [e.g. stock browser, safari]
  - Version: [e.g. 22]
 
+**For whom is this feature?**
```

Comment:
I think this should be a bug question. E.g.

> While [activity] as [persona], I expected [result] but observed [result].

Reply:
And the bold section could say

> **For whom does this bug occur?**

Reply:
Nice! I've made the edit and submitted another pull request. I'm going to close out this one, as I don't know how to edit the settings in a pull request.

Reply:
Ah, no worries. I think you can edit a pull request like before, which is in the main repo switch the branch to this pull request's branch and make the changes and submit it.